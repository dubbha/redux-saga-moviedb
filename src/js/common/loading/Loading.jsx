import React from 'react';
import './style.sass';

const Loading = () => (
  <div className="loading">
    <div className="loading__spinner" />
  </div>
);

export default Loading;
