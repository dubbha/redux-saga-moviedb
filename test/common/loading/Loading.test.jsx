import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from 'common/loading/Loading';

describe('Spinner', () => {
  it('should render successfully', () => {
    const wrapper = shallow(<Loading />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
