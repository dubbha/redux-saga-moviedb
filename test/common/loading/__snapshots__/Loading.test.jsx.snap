// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render successfully 1`] = `
<div
  className="loading"
>
  <div
    className="loading__spinner"
  />
</div>
`;
